store/CSDK/src/CarbonWriter.cpp

```diff
@@ -271,6 +285,22 @@ void CarbonWriter::withBlockletSize(int blockletSize) {
     }
 }
 
+void CarbonWriter::withSchemaFile(char *schemaFilePath) {
```

Comment:
Since this is interface exposed to user, please add comment to describe the function

Reply:
Already describe it in interface:

```
    /**
     * To set the path of carbon schema file
     * @param schemaFilePath The path of carbon schema file
     */
    void withSchemaFile(char *schemaFilePath);
```

Reply:
also added in here